specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -11956,6 +11961,41 @@
       "x-ms-client-name": "BatchDeploymentProperties",
       "additionalProperties": false
     },
+    "BatchDeploymentConfiguration": {
+      "description": "Properties relevant to different deployment types.",
+      "required": [
+        "deploymentConfigurationType"
+      ],
+      "type": "object",
+      "properties": {
+        "deploymentConfigurationType": {
+          "description": "[Required] The type of the deployment",
+          "$ref": "#/definitions/BatchDeploymentConfigurationType"
+        }
+      },
+      "discriminator": "deploymentConfigurationType"
+    },
+    "BatchDeploymentConfigurationType": {
+      "description": "The enumerated property types for batch deployments.",
+      "enum": [
+        "Model",
+        "PipelineComponent"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "BatchDeploymentConfigurationType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Model"
```

Comment:
where is the model with `"x-ms-discriminator-value": "Model"` defined?

Reply:
I just bring the changes of this PR: https://github.com/Azure/azure-rest-api-specs/commit/913cc602374577d6cf2f2fdef4ce3c30a9c46bd4 back.

Reply:
ms-zhenhua This emergency PR is a fixing an error that was introduced by the following PR that was merged 2 days ago.
- https://github.com/Azure/azure-rest-api-specs/pull/28262
If you look at the changes in that PR around lines 11000-12000, you can see that it deletes several definitions related to batch deployments. This should never have been merged, since it is an accidental revert/bash of this PR from last week:
- https://github.com/Azure/azure-rest-api-specs/commit/40a0b442ce796532e2eef43919dd4b4a2df9d7dd

In conclusion, please ignore standard complains, as this PR is merely re-adding changes that have ALREADY been approved, and were accidentally deleted due to a falsely-approved PR.